### DISCUSSION TITLE
Table position search

### DISCUSSION THREAD

**Discussion by JoshWisniewski** [grafana/grafana#46752] [Dashboards, Panels & Visualizations]
First let me say the table filter feature is fantastic.  It's the perfect solution for lots of data sets.

For one of my data sets, it's best to see the row in the context of the other rows that come before and after (which may not match the filter).  Using the filter feature, we can't see the desired row in it's context of the other rows.  A table search feature would be very helpful for finding a specific row in the table and see it in it's context of the surrounding rows.  I've tried using various browser search functionality (Firefox, Chrome, Edge) but all of them can not find content in the table that is not currently on the screen.  Hence, a Grafana table search being ideal.

If possible, it would be great to be able to execute the search from a data link.  For example, I have a dashboard that shows various details extracted from my table.  I'd love to be able to provide a link from my various details dashboard to my full table dashboard with the table automatically scrolled to the search position such that you can see the row in it's context at the click of a link.

Thanks for your consideration,
Josh

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
